jvm/core/deps.bzl

```diff
@@ -15,5 +15,8 @@ main_deps = main_exports + [
 main_runtime_deps = []
 
 # Test dependencies
-test_deps = []
+test_deps = [
+     "//plugins/mocks:jar",
+     "//plugins/reference-assets/jvm:reference-assets",
```

Comment:
Will these flow automatically from the `testutils` dependency?

Reply:
let me try removing these to see if these the flows get loaded still!